templates/pages/admin/form/item_has_conditions_for_new_question_type_modal.html.twig

```diff
@@ -0,0 +1,43 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2025 Teclib' and contributors.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% set modal_aria_label = __("Question has conditions and its type cannot be changed") %}
+{% set modal_name = "new_question_type" %}
+
+{% embed "pages/admin/form/item_has_conditions_modal_base.html.twig" %}
+    {% block modal_headers %}
+        <div class="d-none" data-glpi-form-editor-item-has-conditions-modal-header="question">
+            <h3 class="m-0">{{ __("This question is used in the conditions of other form elements with a value operator that is not available with the new type") }}</h3>
```

Comment:
I think a small margin at the bottom would be good, right now the text seems super close to the title.